generated/nitclk/nitclk/session.py

```diff
@@ -73,13 +76,13 @@ class SessionReference(object):
     Default Value is empty string
     '''
     pause_trigger_master_session = _attributes.AttributeSessionReference(6)
-    '''Type: nimi-python Session class, nitclk.SessionReference, NI-TClk Session Number
+    '''Type: nimi-python Session class or nitclk.SessionReference
```

Comment:
* I wonder if "nimi-python Session class" makes sense to users.
* Why have "class" there but not after `nitclk.SessionReference`?

Not sure best way to reword. Maybe just "Driver Session or nitclk.SessionReference" ?

Reply:
I don't have a better way either so I will just use "Driver Session or nitclk.SessionReference"